scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -1963,6 +1964,12 @@ function Set-ListenConnectUriForLinuxContainers {
     $env:IOTEDGE_HOST = "${listenAddress}:15580"
 }
 
+function Disable-IoTEdgeMoby {
+    Invoke-Native "sc config iotedge depend= """"" 
```

Comment:
Use `'` for strings without interpolation, which will also make it so you don't have to escape the `"`.

Same for the next line.